hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -213,6 +216,7 @@ public List<HapiSpec> getSpecsInSuite() {
     }
 
     @HapiTest
+    @Tag("crypto")
```

Comment:
I think we should delete this tag since we've tagged the whole class

Reply:
Good catch! Leftover from an early version. Will remove it.